src/nft-price-checker.go

```diff
@@ -157,37 +169,45 @@ func ethPrice() float64 {
 }
 
 // Fetching prices from the results of the API to build the final string
-func getResult(inputCollections []Collection, outputType string) string {
+func computeEstimates(inputCollections []Collection) Estimates {
 	ethSum := 0.0
-	result := ""
-	totalResult := ""
+	var estimates Estimates
+	collectionEstimates := make(map[string]CollectionEstimate)
 	for _, inputCollection := range inputCollections {
-		inputCollectionFloorPrice := floorPrice(inputCollection.CollectionID)
-		inputCollectionNumberOfAssets := inputCollection.Count
-		ethCollectionTotalEstimate := (inputCollectionFloorPrice * inputCollectionNumberOfAssets)
-		ethSum += ethCollectionTotalEstimate
-		if outputType == "web2" {
-			if inputCollectionFloorPrice == 0 {
-				result += fmt.Sprintf("  x %s cannot be found on Opensea \n", inputCollection.CollectionID)
+		floorPrice := floorPrice(inputCollection.CollectionID)
+		ethEstimate := (floorPrice * inputCollection.Count)
+
+		collectionEstimates[inputCollection.CollectionID] = CollectionEstimate{floorPrice, inputCollection.Count, ethEstimate}
+
+		ethSum += ethEstimate
+	}
+	estimates.CollectionEstimates = collectionEstimates
+	estimates.EthTotalEstimate = ethSum
+
+	return estimates
+}
+
+func presentEstimates(estimates Estimates, outputType string) string {
+	usdEstimate := ethPrice() * estimates.EthTotalEstimate
+	if outputType == "web2" {
+		result := ""
+		for collectionID, collectionEstimates := range estimates.CollectionEstimates {
+			if collectionEstimates.FloorPrice == 0 {
+				result += fmt.Sprintf("  x %s cannot be found on Opensea, or its floor price is equal to 0\n", collectionID)
 			} else {
-				result += fmt.Sprintf("--> %s Floor price = %f eth\n", inputCollection.CollectionID, inputCollectionFloorPrice)
-				result += fmt.Sprintf(" So %f*%f=%f eth\n", inputCollectionNumberOfAssets, inputCollectionFloorPrice, ethCollectionTotalEstimate)
+				result += fmt.Sprintf("--> %s Floor price = %f eth\n	So %f*%f=%f eth\n",
```

Comment:
```suggestion
				result += fmt.Sprintf("--> %s Floor price = %f eth\nSo %f*%f=%f eth\n",
```
If you think keeping a white space, would this work with `\t` character instead?

Reply:
thanks I didn't know this character 👍🏼